### ISSUE TITLE
Offset Bug in 0.6.0

### ISSUE THREAD

**Issue by Skiid** [vandeseer/easytable#62]
When i draw a repeated header table the endY gets shifted lower with every page the table gets drawn on. The vice versa effect happens with the header offset inside the .draw() function of the RepeatedHeaderTableDrawer (the table gets shifted higher every time the table reaches a new page).

```
private fun drawTable(
        table: Table,
        tableRect: PDRectangle,
        pdfDocument: PDDocument,
        contentStream: PDPageContentStream
    ) {

        RepeatedHeaderTableDrawer.builder()
            .table(table)
            .startY(tableRect.upperRightY)
            .startX(tableRect.lowerLeftX)
            .contentStream(contentStream)
            .endY(footerOffset)
            .build()
            .draw(Supplier<PDDocument> { pdfDocument }, Supplier<PDPage> { PDPage(PDRectangle.A4) }, headerOffset)
    }
```

On version 0.5.2 the offsets are working correctly.

**Comment by Skiid** [comment]
The endY seems to work as expected but the headerOffset is still buggy

**Comment by vandeseer** [comment]
Thanks for reporting @Skiid! Do you have a minimum working example where one can reproduce the problem?

**Comment by vandeseer** [comment]
I cannot reproduce the issue. I will close this issue now. Please reopen it again if you have a minimum working example where one can reproduce the issue. Thanks.

**Comment by vandeseer** [comment]
I see now what the problem is: the first row gets drawn above the rest of the table. That's a deviation from earlier versions.

**Comment by nellymin** [comment]
For a quick workaround you could add header.getHeight() to the value of yOffset

**Comment by vandeseer** [comment]
I will fix this in the next version. You can also look at this code here and use a custom class for the time being: 

- https://gist.github.com/vandeseer/8c2ca07265dc44873eeb5ead64b37dd7

I hacked this together for https://github.com/vandeseer/easytable/issues/70. So in the next release I will include the fix as well as a feature to set how many rows should be repeated.

**Comment by vandeseer** [comment]
This issue should now be fixed with version 0.6.2. 

Thanks for reporting again @Skiid and thanks for helping out with a workaround @nellymin.
